pkg/cmd/root.go

```diff
@@ -27,6 +27,7 @@ func Execute() {
 		API(),
 		Card(),
 		Toolbox(),
+    CI(),
```

Comment:
Very minor but the spacing on this doesn't look happy

Reply:
pressed CTRL+S and fixed this!